pygsti/tools/jamiolkowski.py

```diff
@@ -273,6 +273,27 @@ def fast_jamiolkowski_iso_std_inv(choi_mx, op_mx_basis):
     #transform operation matrix into appropriate basis
     return _bt.change_basis(opMxInStdBasis, op_mx_basis.create_equivalent('std'), op_mx_basis)
 
+def sum_of_negative_choi_eigenvalues_gate(op_mx, op_mx_basis):
+    """
+    Compute the sum of the negative Choi eigenvalues of a process matrix.
+
+    Parameters
+    ----------
+    op_mx : np.array
+
+    op_mx_basis : Basis
+
+    Returns
+    -------
+    float
+        the sum of the negative eigenvalues of the Choi representation of op_mx
+    """
+    sumOfNeg = 0
+    J = fast_jamiolkowski_iso_std(op_mx, op_mx_basis)  # Choi mx basis doesn't matter
+    evals = _np.linalg.eigvals(J)  # could use eigvalsh, but wary of this since eigh can be wrong...
```

Comment:
Please elaborate on what you noticed when using ``eigh`` instead of ``eig``. I'd consider the former function more reliable than the latter.

Did you just copy over the comment from ``sums_of_negative_choi_eigenvalues`` which predates this PR?

Reply:
This one I can confirm is a carry over comment from Erik's original code, but I don't know what he meant by this.

Reply:
Ping @enielse. Do you remember why you preferred eig as opposed to eigh?

Reply:
I opened issue #582 to investigate this. Marking this issue as resolved so we can get this merged.